src/libcrun/cgroup.c

```diff
@@ -2828,6 +2828,26 @@ update_cgroup_v2_resources (runtime_spec_schema_config_linux_resources *resource
         return ret;
     }
 
+  /* Write unified resources if any.  They have higher precedence and override any previous setting.  */
+  if (resources->unified)
+    {
+      size_t i;
+
+      for (i = 0; i < resources->unified->len; i++)
+        {
+          size_t len;
+
+          if (strchr (resources->unified->keys[i], '/'))
```

Comment:
We should map the keys to the known controllers as well and fail if it isn't known to the runtime.

Reply:
why would we need to do that?  I think we can just write to the specified file (and that was one of the reasons for using the map in the OCI spec PR), since the key is the file name itself.  In any case the OCI runtime is not doing any handling of this information